adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,175 @@ endif::[]
+--set sizing.uaa.count=2 --set sizing.mysql.count=1
+helm upgrade susecf-scf --namespace scf suse/cf --reuse-values --set config.HA=false \ 
+--set=sizing.diego_cell.count=3 --set=sizing.adapter.count=2 --set=sizing.api_group.count=2 \ 
+--set=sizing.autoscaler_actors.count=2 --set=sizing.autoscaler_api.count=2 \ 
+--set=sizing.autoscaler_metrics.count=2 --set=sizing.cc_clock.count=2 --set=sizing.cc_uploader.count=2 \ 
+--set=sizing.cc_worker.count=2 --set=sizing.cf_usb_group.count=2 --set=sizing.diego_api.count=2 \ 
+--set=sizing.diego_brain.count=2 --set=sizing.diego_ssh.count=2 --set=sizing.doppler.count=2 \ 
+--set=sizing.locket.count=2 --set=sizing.log_api.count=2 --set=sizing.log_cache_scheduler.count=2 \ 
+--set=sizing.nats.count=2 --set=sizing.nfs_broker.count=2 --set=sizing.router.count=2 \ 
+--set=sizing.routing_api.count=2 --set=sizing.syslog_scheduler.count=2 --set=sizing.tcp_router.count=2 \ 
+--set=sizing.mysql.count=1
+----
++
+Delete the PVC(s) for `mysql-1` from both `uaa` and `scf` namespaces but leave the associated PV(s) as is.
+
+.. Run the upgrade to 1.5 but keep the `mysql` at 1 for now, leaving the rest of the install as HA.
++
+[source,bash]
+----
+helm upgrade susecf-uaa --namespace uaa suse/uaa  -f <values.yaml> --set config.HA=true \ 
+--set config.HA_strict=false --set sizing.mysql.count=1
+helm upgrade susecf-scf --namespace scf suse/cf -f <values.yaml> --set config.HA=true \ 
+--set config.HA_strict=false --set sizing.mysql.count=1 --set "secrets.UAA_CA_CERT=${CA_CERT}"
+----
++
+.. Upgrade the `mysql` role to the default HA configuration, bringing 1.5 to full HA.
++
+[source,bash]
+----
+helm upgrade susecf-uaa --namespace uaa suse/uaa -f <values.yaml> --set config.HA_strict=true \ 
+--set config.HA=true
+helm upgrade susecf-scf --namespace scf suse/cf -f <values.yaml> --set config.HA_strict=true \ 
+--set config.HA=true --set "secrets.UAA_CA_CERT=${CA_CERT}"
+----
+
+IMPORTANT: If you are using a buildpack that uses the same name as a shipped buildpack, you will need to rename it to a unique name. Based on our existing model of stackless buildpacks, any buildpack name already in use is considered reserved. 
+
+IMPORTANT: As of {scf} 2.18.0, since our `cf-deployment` version is 9.5, the `cflinuxfs2` stack is no longer supported, as was advised in {scf} 2.17.1 or {cap} 1.4.1. The `cflinuxfs2` buildpack is no longer shipped, but if you are upgrading from an earlier version, `cflinuxfs2` will not be removed. However, for migration purposes but we encourage all admins to move to `cflinuxfs3` or `sle15` as newer buildpacks will not work with the deprecated `cflinuxfs2`. If you still want to use the older stack, you will need to build an older version of a buildpack to continue for the app to work, but you will be unsupported. (If you are running on `sle12`, we will be retiring that stack in a future version so start planning your migration to `sle15`.)
```

Comment:
Extra `but` in `However, for migration purposes but we encourage all`

Reply:
but gone :)